jetstream/config.py

```diff
@@ -363,28 +422,36 @@ class MetricDefinition:
     friendly_name: Optional[str] = None
     description: Optional[str] = None
     bigger_is_better: bool = True
+    analysis_basis: mozanalysis.experiment.AnalysisBasis = None
 
     def resolve(self, spec: "AnalysisSpec", experiment: ExperimentConfiguration) -> List[Summary]:
         if self.select_expression is None or self.data_source is None:
             # checks if a metric from mozanalysis was referenced
             search = experiment.platform.metrics_module
-            metric = _lookup_name(
+            mozanalysis_metric = _lookup_name(
                 name=self.name,
                 klass=mozanalysis.metrics.Metric,
                 spec=spec,
                 module=search,
                 definitions={},
             )
+            metric = Metric.from_mozanalysis_metric(
+                mozanalysis_metric=mozanalysis_metric,
+                analysis_basis=self.analysis_basis
+                or mozanalysis.experiment.AnalysisBasis.ENROLLMENTS,
```

Comment:
The default is to use enrollments for analysis by default. Based on the proposal it sounds like exposures should become the default at some point. I guess that would break current assumptions of the results and likely requires some communication to parties that use the results.